releaseNotes/AzureSpringCloudAppConfigurationConfig.md

```diff
@@ -0,0 +1,38 @@
+# azure-spring-cloud-appconfiguration-config
+
+[Source code][source_code] | [Package (Maven)][package] | [README][readme] | [Product documentation][docs] | [Samples][samples]
+
+# azure-spring-cloud-appconfiguration-config-web
+
+[Source code web][source_code_web] | [Package (Maven) web][package_web] | [Product documentation][docs]
+
+## 2.0.0-beta.1 - May 5, 2021
+
+* Enables loading of configurations from multiple config stores.
+* Each store can load configurations by using key/label `selects`. Many `selects` can be used, default is key = `/applicaiton/*` with label = `${spring.profiles.active}` or if null `\0`. See [README][readme] for full configuration.
+* Authentication vis Connection String, Managed Identity, or any method supported by Azure Identity SDK, see Token Credential Provider in README.
+* Each individual store can have monitoring enabled and disabled and have a cache-expiration time set.
+  * Monitoring requires at least one trigger to be set. `spring.cloud.azure.appconfiguration.stores[0].monitoring.triggers[0].key` is required `spring.cloud.azure.appconfiguration.stores[0].monitoring.triggers[0].label` is optional.
+  * Monitoring has both Pull and Push model options.
+    * Pull checks the config store for changes based on Activity when the cache has expired.
+    * Push involves setting up a Azure Event Grid Web Hook to notify the client application that a change has occurred. Two Spring Actuator Endpoints exist to connect to. `appconfiguration-refresh` triggers the cache to reset on configurations on an application. `appconfiguration-refresh-bus` triggers a refresh on all instances subscribed to the same Service Bus.
+      * Push refresh requires a token name and value to be set for verification. The application needs to be running when creating the Web Hook as a response needs to be returned to setup a Web Hook.
+* Feature Flag loading can be enabled per config store `spring.cloud.azure.appconfiguration.stores[0].feature-flags.enable`
+  * A single label can be used to load feature flags, default `\0` i.e. (No Label).
```

Comment:
Not sure what you are trying to convey here. We don't support labels for feature flags and we support it now?